src/stavetie.ts

```diff
@@ -69,26 +110,23 @@ export class StaveTie extends Element {
     }
 
     // Success. Lets grab 'em notes.
-    this.first_note = notes.first_note;
-    this.first_indices = notes.first_indices;
-    this.last_note = notes.last_note;
-    this.last_indices = notes.last_indices;
+    this.notes = notes;
```

Comment:
This is okay, do the regression tests show any diffs?

Reply:
No diffs, the information was simply duplicated.